examples/nanoChatGPT/train_rlhf.py

```diff
@@ -10,24 +9,49 @@
 from shared import setup
 from tensordict.nn import set_skip_existing, TensorDictModuleBase
 from torch import vmap
-
+from torchrl.collectors import SyncDataCollector
 from torchrl.data import LazyTensorStorage, TensorDictReplayBuffer
 from torchrl.data.replay_buffers import SamplerWithoutReplacement
 from torchrl.objectives import ClipPPOLoss
 from torchrl.objectives.value import GAE
+from torchrl.envs import set_exploration_type, ExplorationType
 from utils import load_and_update_config
+import tqdm
+import csv
+import tiktoken
+from copy import deepcopy
 
 HERE = Path(__file__).parent
 
+def append_to_csv(number, filename):
```

Comment:
a terrible way to log rewards on the fly

Reply:
Why do this instead of using`logging` or similar?